src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    <ul>
+      {todos.map(todo => (
+        <li key={todo.id}>
+          <TodoInfo todo={todo} />
+        </li>
```

Comment:
you don't need `li` wrapper
```suggestion
        <TodoInfo todo={todo} key={todo.id} />
```